pandas/tests/reshape/test_pivot.py

```diff
@@ -1266,21 +1266,18 @@ def test_crosstab_normalize(self):
                                            [0.25, 0.75],
                                            [0.4, 0.6]],
                                           index=pd.Index([1, 2, 'All'],
-                                                         name='a',
-                                                         dtype='object'),
```

Comment:
Ah, I see down below where it does change from object. I think that's ok though.